docs/kyma/05-01-overview.md

```diff
@@ -3,55 +3,81 @@ title: Overview
-One of the installation artifacts is the Kyma Installer image that contains all component charts located in the `kyma/resources` folder. Both cluster and local installation files contain a full list of these component names and their Namespaces.
-Components that are not an integral part of the default Kyma Lite package are preceded with hashtags (#). It means that the Installer skips them during installation.   
-You can customize component installation files by:
-- Removing hashtags in front of the component entries to enable a component installation.
-- Adding hashtags to disable a component installation.
-- Adding new components to the list along with their chart definition in the `kyma/resources` folder. In that case you must create your own [Kyma Installer image](#installation-use-your-own-kyma-installer-image) as you are adding new chart configuration.
+One of the released Kyma artifacts is the Kyma-Installer, a Docker image that combines the Installer executable with charts for all components available in the release.
+The Kyma-Installer can install only the components contained in its image.
+**Installation** Custom Resource specifies which components of all available ones should be actually installed.
+Components that are not an integral part of the default Kyma Lite package are commented out with a hash character (#) and skipped during installation.
+You can customize list of components used for installation by:
+- Uncommenting component entry to enable a component installation.
+- Commenting out an entry using hash chacter to disable a component installation.
+- Adding new components to the list along with their chart definition. In that case you must create your own [Kyma-Installer image](#installation-use-your-own-kyma-installer-image) as you are adding new component to Kyma.
 
 For more details on custom component installation, see [this](#configuration-custom-component-installation) document.
 
 ### Overrides
 
-The configuration files provide production-ready settings for the Installer. These include ConfigMaps and Secrets with the overrides for the values hardcoded in the `values.yaml` files of component charts.
-
-You can modify local settings, such as memory limits for a given resource, by changing values in the existing ConfigMaps and Secrets or adding new ones. Every ConfigMap and Secret that the Installer reads must contain [obligatory labels](#configuration-helm-overrides-for-kyma-installation-user-defined-overrides).
-
+Common overrides that affect entire installation, for example `global.domainName`, are already described in the [installation procedure](#installation-overview).
+Other overrides are component-specific.
+To learn about configuration options for a given component, refer to **Configuration** section of the component's documentation.
+Once you know the name and possible set of values for a configuration option, define an override for it by creating a ConfigMap or a Secret. You can also extend an existing one.
```

Comment:
This sentence is redundant IMO, I'd remove it.

Reply:
Done.